tensorboard/BUILD

```diff
@@ -66,6 +66,7 @@ py_library(
     deps = [
         "//tensorboard:expect_tensorflow_installed",
         "//tensorboard/plugins/audio:audio_plugin",
+        "//tensorboard/plugins/beholder/server_side:beholder_plugin",
```

Comment:
Hm, having this server_side directory makes beholder a little different from all the other plugins.  Maybe we should move beholder_plugin.py up into the top-level directory for consistency with the other plugins, where both "server side" code (the plugin.py) and "API" code (the summary.py) coexist in one directory.